x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/routes/signals/query_signals_route.test.ts

```diff
@@ -25,7 +25,7 @@ describe('query for signal', () => {
   context.core.elasticsearch.client.asCurrentUser.search.mockResolvedValue(
     elasticsearchClientMock.createSuccessTransportRequestPromise(getEmptySignalsResponse())
   );
-  const ruleDataClient = ruleRegistryMocks.createRuleDataClient('.alerts-security.alerts-');
+  const ruleDataClient = ruleRegistryMocks.createRuleDataClient('.alerts-security.alerts');
```

Comment:
Because of these changes in mock object https://github.com/elastic/kibana/pull/222584/files#r2126962190